ResearchKit/Localized/en.lproj/Localizable.strings

```diff
@@ -86,6 +86,10 @@
 /* Alert for invalid email address value */
 "INVALID_EMAIL_ALERT_MESSAGE" = "Invalid email address: %@";
 
+/* Alerts for invalid location coordinates */
+"LOCATION_ADDRESS" = "Enter an address";
+"LOCATION_ERROR_TITLE" = "Location error";
```

Comment:
Can we give this title a better wording, such as 
`Failed to resolve specified address`? 
`Unresolvable location`?
`Unresolvable address`?